t/14-isotovideo.t

```diff
@@ -394,6 +394,14 @@ subtest 'publish assets' => sub {
         is $return_code, 0, 'Successful handle_generated_assets' or die path(bmwqemu::STATE_FILE)->slurp;
         ok !-e $publish_asset, 'test.qcow2 does not exist';
     };
+
+    subtest 'prevent upload assets when publish_hdd is none' => sub {
+        $bmwqemu::vars{PUBLISH_HDD_1} = 'none';
```

Comment:
There should be another case using `NONE` or `None`

Reply:
that would go a bit too far. This is really just a trivial change costing too much already. Just keep as is

Reply:
Done